cms/static/sass/bootstrap/studio-main.scss

```diff
@@ -3,7 +3,7 @@
 // -----------------------------
 
 // Bootstrap theme
-@import 'bootstrap/theme';
+@import 'cms/bootstrap/theme';
```

Comment:
The challenge that @HarryRein and I discovered is that Studio includes the LMS Sass directory on its import path. I think this is to support Studio rendering XBlocks with the correct styles, but it means that there are confusing situations where the LMS file loads instead of the CMS one. The solution we came up with was to add this extra directory as essentially a namespace to avoid collisions.